src/core/lib/security/transport/secure_endpoint.cc

```diff
@@ -91,9 +107,13 @@ struct secure_endpoint {
   /* saved handshaker leftover data to unprotect. */
   grpc_slice_buffer leftover_bytes;
   /* buffers for read and write */
-  grpc_slice read_staging_buffer = GRPC_SLICE_MALLOC(STAGING_BUFFER_SIZE);
-  grpc_slice write_staging_buffer = GRPC_SLICE_MALLOC(STAGING_BUFFER_SIZE);
+  grpc_slice read_staging_buffer ABSL_GUARDED_BY(read_mu) = grpc_empty_slice();
```

Comment:
we're initializing in the constructor, so no need to also do that here (and as a bonus, if we change the constructor and forget these, it's gonna raise an MSAN error)

Reply:
Done